packages/instant/src/redux/reducer.ts

```diff
@@ -1,16 +1,21 @@
+import { BuyQuote } from '@0xproject/asset-buyer';
 import { BigNumber } from '@0xproject/utils';
 import * as _ from 'lodash';
 
-import { Action, ActionTypes } from '../types';
+import { Action, ActionTypes, AsyncProcessState } from '../types';
 
 export interface State {
```

Comment:
add state for `Provider`, and other things required to remove hard-coded values.

Reply:
Not going to do this for now. Should be done when we implement different wallets.